ComputerPlayer.java

```diff
@@ -16,8 +17,9 @@ public static int randInt(int min, int max) {
 
     @Override
     public Integer nextMove(Board board) {
-        int index = randInt(0, board.availablePositions().size());
-        return (int) board.availablePositions().get(index);
+        ArrayList availiblePositions = board.availablePositions();
+        int index = randInt(2, availiblePositions.size());
+        return (int) availiblePositions.get(index - 1);
```

Comment:
Any way you can get a random integer in the 0 indexed range rather than have to reduce it by 1?
Spelling mistake on availablePositions variable too.